python/addons/embed_tlm_pytorch.py

```diff
@@ -231,11 +308,13 @@ def _model_mask(self, nctx):
             return subsequent_mask(nctx)
 
     def forward(self, x):
-        # the following line masks out the attention to padding tokens
+        # the following line masks out the attention to padding tokens: Bx1x1xT
```

Comment:
Normally we annotate shapes as `[B, 1, 1, T]` a consistent shape just makes things easier to read